modules/registry-authentication.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * registry/registry-options.adoc
+
+[id="authentication_{context}"]
+= Authentication
+{product-title} can communicate with registries to access private image
+repositories using credentials supplied by the user. This allows {product-title}
+to push and pull images to and from private repositories.
+
+[id="registry-authentication_{context}"]
+== Registry authentication with Podman
+Some container images registries require access authorization. Podman is an open source tool for managing containers and container images and interacting with image registries. You can use podman to authenticate your credentials, pull the registry image, and store local images in a local file-system. The following is a generic example of authenticating the registry with podman. You can use the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog] to search for specific container images from the Red Hat Repository. Select the image you need. Then click the *Get this image* tab to find the podman command for your container image.
+
+
+. Login using the `podman login` command and your username and password authentication to the registry:
++
+[source,terminal]
+----
+$ podman login registry.redhat.io
+ Username:<your_registry_account_username>
+ Password:<your_registry_account_password>
+----
+
+. Download the image and save it locally using the `podman pull` command:
++
+[source,terminal]
+----
+$ podman pull registry.redhat.io/<registry_image_name>
```

Comment:
Maybe better s/<registry_image_name>/<repository_name>

Since in https://catalog.redhat.com/software/containers/rhel8/ruby-25/5ba0ae3bbed8bd6ee81985ea?container-tabs=gti&gti-tabs=registry-tokens , it's `Repository: rhel8/ruby-25`